src/core/package/index.ts

```diff
@@ -215,8 +217,9 @@ export class Package {
    */
   async writePackage() {
     const pkgPath = this.resolvePath('package.json')
+    const indent = detectIndent(readFileSync(pkgPath, 'utf8')).indent || '  '
```

Comment:
Ideally I'd like to detect this when the package is initialised as that's when we read in the package.json.

Reply:
Ok, and then store it on a property to be used there again? Happy to update

Reply:
Exactly 👍

Reply:
efe77dfe771c9d3d819a3c0ca4eed7ce5628b50a should be good :)

I doubled check what we might be missing out on not using [readJSONSync](https://github.com/jprichardson/node-jsonfile/blob/master/index.js#L40-L61), we're not using the "fancy" options, and we're enforcing `utf8` so from my understanding `stripBom` doesn't help us with anything(?)

Let me know :)